docs/html/reference/installation-report.md

```diff
@@ -0,0 +1,174 @@
+# Installation Report
+
+The `--report` option of the pip install command produces a detailed JSON report of what
+it did install (or what it would have installed, if used with the `--dry-run` option).
+
+## Specification
+
+The report is a JSON object with the following properties:
+
+- `version`: the string `0`, denoting that the installation report is an experimental
+  feature. This value will change to `1`, when the feature is deemed stable after
+  gathering user feedback (likely in pip 22.3 or 23.0). Backward incompatible changes
+  may be introduced in version `1` without notice. After that, it will change only if
+  and when backward incompatible changes are introduced, such as removing mandatory
+  fields or changing the semantics or data type of existing fields. The introduction of
+  backward incompatible changes will follow the usual pip processes such as the
+  deprecation cycle or feature flags. Tools must check this field to ensure they support
+  the corresponding version.
+
+- `pip_version`: a string with the version of pip used to produce the report.
+
+- `install`: an array of `InstallationReportItem` (see below) representing the
+  distribution packages (to be) installed.
+
+- `environment`: an object describing the environment where the installation report was
+  generated. See [PEP 508 environment
+  markers](https://peps.python.org/pep-0508/#environment-markers) for more information.
+  Values have a string type.
+
+An `InstallationReportItem` is an object describing a (to be) installed distribution
```

Comment:
```suggestion
(InstallationReportItem)=

An `InstallationReportItem` is an object describing a (to be) installed distribution
```

Combined with the above comment, this would add a meaningful link.